includes/class-newspack-image-credits.php

```diff
@@ -322,14 +317,20 @@ function( $matches ) use ( $placeholder_image, $size ) {
 	 * @return array Filtered array of image data.
 	 */
 	public static function maybe_show_placeholder_image( $image_data, $attachment_id, $size, $icon ) {
-		if ( ! is_singular() && ! is_archive() ) {
```

Comment:
I'm not sure why I had this logic at first. It prevents the placeholder image from appearing in search results.